### PR TITLE
Allow TestSuite Specific Bootstraps

### PR DESCRIPTION
**Pull request by anubisthejackle**
This PR attempts to implement #3795 

Tests have been created to verify that the configuration parses as expected, and that bootstraps are loaded both in a standard flow, as well as in process isolation.

### CODE REVIEW

File: src/Util/PHP/Template/TestCaseMethod.tpl [hunk 1]
```diff
@@ -122,5 +122,14 @@ if (isset($GLOBALS['__PHPUNIT_BOOTSTRAP'])) {
     require_once $GLOBALS['__PHPUNIT_BOOTSTRAP'];
     unset($GLOBALS['__PHPUNIT_BOOTSTRAP']);
 }
+var_dump( $GLOBALS );
```
Comment (sebastianbergmann) [inline src/Util/PHP/Template/TestCaseMethod.tpl:125]: This looks like a leftover from debugging.

File: src/Util/PHP/Template/TestCaseMethod.tpl [hunk 2]
```diff
@@ -122,5 +122,14 @@ if (isset($GLOBALS['__PHPUNIT_BOOTSTRAP'])) {
...
+if (isset($GLOBALS['__PHPUNIT_TESTSUITE_BOOTSTRAPS'])) {
+    var_dump( $GLOBALS['__PHPUNIT_TESTSUITE_BOOTSTRAPS']);
```
Comment (sebastianbergmann) [inline src/Util/PHP/Template/TestCaseMethod.tpl:127]: This looks like a leftover from debugging.

Comment (anubisthejackle) [reply src/Util/PHP/Template/TestCaseMethod.tpl]: Good eye. Missed those.

**Comment by anubisthejackle** [comment]
@sebastianbergmann Just checking in to see if there's anything that you'd like to do for this.

**Comment by sebastianbergmann** [comment]
> @sebastianbergmann Just checking in to see if there's anything that you'd like to do for this.

Nothing right now except time to actually look into this. I want to finish the migration from `TestListener` and `ResultPrinter` to the new event system first before I look at other things, sorry.

**Comment by anubisthejackle** [comment]
> I want to finish the migration from `TestListener` and `ResultPrinter` to the new event system first before I look at other things, sorry.

Makes sense to me. I'm not in any rush.

**Comment by sebastianbergmann** [comment]
Thank you for your contribution. I appreciate the time you invested in preparing this pull request. However, I have decided not to merge it.

**Pull request closed without merging**